### ISSUE TITLE
Accuracy on the CAR dataset

### ISSUE THREAD

**Issue by Thea1** [TACJu/TransFG#13]
To the best of my own ability, I can only achieve up to 90% accuracy on the car dataset. Is there something wrong with me?  I would like to ask if the parameters of the training car dataset are set the same as the cub dataset?

**Comment by Markin-Wang** [comment]
Also, I run the ViT on cars dataset with batchsize 8, 20000 steps. The accuracy is 90.86, much less than the 93.7 reported in the paper. Waiting for the author to release the code for training the cars datasets.

**Comment by 2387180967qqcom** [comment]
have you solved the problem？I also had this problem

**Comment by wywd** [comment]
> 

Hi. Have you solved this problem? I have the same problem. It's very troublesome!!!

**Comment by wywd** [comment]
> Also, I run the ViT on cars dataset with batchsize 8, 20000 steps on GPU cards. The accuracy is 90.86, much less than the 93.7 reported in the paper. Waiting for the author to release the code for training the cars datasets.

Hi. Have you solved this problem? I have the same problem. It's very troublesome!!!

**Comment by Markin-Wang** [comment]
> 

No, I finally cannot achieve the reported results on CUB, Dogs and Car datasets. More importantly, there are a large disparities. Therefore, I have to include both the reported results and my reimplemented results on my paper on dog dataset and abadon the Car dataset. This really imposed a negative influence on during the paper review, but fortunatelly my paper was accpeted finally. We are still waiting for the response from authors. But at this stage, maybe you can consider to change another backbone and report the reimplemented results of transFG.

**Comment by Flying-farmer** [comment]
I cannot find the file  'cars_test_annos_withlabels.mat '  , can anybody help me?

**Comment by Flying-farmer** [comment]
I cannot find the file  'cars_test_annos_withlabels.mat '  , can anybody help me?

**Comment by hongbo-sun** [comment]
is there anyone have achieved the reported results on car? can u share the training setting？
